nano/node/rocksdb/rocksdb.cpp

```diff
@@ -96,15 +96,86 @@ nano::rocksdb::store::store (nano::logger_mt & logger_a, boost::filesystem::path
 	nano::set_secure_perm_directory (path_a, error_chmod);
 	error = static_cast<bool> (error_mkdir);
 
-	if (!error)
+	if (error)
+	{
+		return;
+	}
+
+	generate_tombstone_map ();
+	small_table_factory.reset (::rocksdb::NewBlockBasedTableFactory (get_small_table_options ()));
+
+	// TODO: get_db_options () registers a listener for resetting tombstones, needs to check if it is a problem calling it more than once.
```

Comment:
Good thing to be reviewed.

Reply:
Encapsulating this to only be called once looks correct.